REPLICATION_POLICIES.md

```diff
@@ -0,0 +1,293 @@
+Replication policies work quite differently on Android from Java. Please see the relevant section of this guide 
```

Comment:
You need your h1 above this, "Replication Policies" with the other headings shifted down a level.

Otherwise it's too easy to skip over this important sentence.